pygeos/test/test_creation.py

```diff
@@ -298,13 +298,27 @@ def test_create_collection_wrong_geom_type(func, sub_geom):
 
 def test_box():
     actual = pygeos.box(0, 0, 1, 1)
-    assert str(actual) == "POLYGON ((1 0, 1 1, 0 1, 0 0, 1 0))"
+    assert str(actual) == "POLYGON ((0 0, 1 0, 1 1, 0 1, 0 0))"
```

Comment:
This is an API change that might be breaking to some applications. In the previous implementation it followed shapely (see here: https://shapely.readthedocs.io/en/stable/manual.html#shapely.geometry.box)

Did you have any reason to adapt the order of coordinates?

Reply:
I don't know the exact reason for shapely's behaviour, but I think returning the normalized version also makes sense:

```
In [3]: pygeos.normalize(pygeos.Geometry("POLYGON ((1 0, 1 1, 0 1, 0 0, 1 0))"))
Out[3]: <pygeos.Geometry POLYGON ((0 0, 0 1, 1 1, 1 0, 0 0))>
```

Although that's still something else ..

Reply:
Hmm, the output of normalize is actually no longer counter-clock-wise. I didn't expect that normalize would change that (at least not change to *not* be CCW).

BTW, Shapely has a `ccw=True` keyword (so which we should still add at some point for compatibility with shapely, unless we deprecate it there)

Reply:
Whoops, right, normalized is clockwise.

No particular reason to change coordinate order other than was trying to start from the bottom left (minimum) coordinate similar to normalize.

Can easily change back to match shapely, no compelling reason to diverge from prior precedent here if that might break people.  Will also look into adding `ccw=True` parameter.